packages/toolpad-app/src/toolpadDataSources/local/server.ts

```diff
@@ -78,7 +78,7 @@ function formatCodeFrame(location: esbuild.Location): string {
 }
 
 async function createMain(): Promise<string> {
-  const relativeFunctionsFilePath = [`.`, getFunctionsFile('.')].join(path.sep);
+  const relativeFunctionsFilePath = path.resolve([`.`, getFunctionsFile('.')].join(path.sep));
```

Comment:
Why can't we use relative paths on windows? I intended it to be relative as to keep the file portable.

Reply:
I just tried to fix the issues at first, didn't dig enough to understand them 100% yet or see if there's a better solution so I'll still do that.

Reply:
Oh, I think I understand now. On windows it's normalizing the path with backslashes and that makes it invalid for node.js module identifiers. I think we should do something along the lines of

```tsx
function pathToNodeImportSpecifier(importPath: string): string {
  const normalized = path.normalize(importPath).split(path.sep).join('/');
  return normalized.startsWith('/') ? normalized : `./${normalized}`;
}
```